utils/ci-test.sh

```diff
@@ -1,31 +1,42 @@
 #!/bin/bash
 
+# SKIP_LINT=true - only run the tests
+
 set -e
 
+MATRIX=(
+	""
+	"-tags coprocess"
+)
+
 # print a command and execute it
 show() {
-	echo "$@"
+	echo "$@" >&2
```

Comment:
Why not? The tools we run use stdout and stderr, so it doesn't make much of a difference. This was for consistency with fatal, mainly.